simpleclient/src/main/java/io/prometheus/client/CollectorRegistry.java

```diff
@@ -105,15 +105,32 @@ public void clear() {
     List<String> names = new ArrayList<String>();
     for (Collector.MetricFamilySamples family : mfs) {
       switch (family.type) {
+        case COUNTER:
+          names.add(family.name + "_total");
+          names.add(family.name + "_created");
+          names.add(family.name);
+          break;
         case SUMMARY:
           names.add(family.name + "_count");
           names.add(family.name + "_sum");
+          names.add(family.name + "_created");
           names.add(family.name);
           break;
         case HISTOGRAM:
           names.add(family.name + "_count");
           names.add(family.name + "_sum");
           names.add(family.name + "_bucket");
+          names.add(family.name + "_created");
+          names.add(family.name);
+          break;
+        case GAUGE_HISTOGRAM:
```

Comment:
Believe gauge histogram should also have `_created`?

Reply:
No, gauges don't have a created.

Reply:
If so, should we change the proto then? HistogramValue can be used with GAUGE_HISTOGRAM:
https://github.com/OpenObservability/OpenMetrics/blob/master/proto/openmetrics_data_model.proto#L136-L138

I suppose implementors could enforce that's not combined together?

Reply:
We should clarify that then.